PyRoute/Galaxy.py

```diff
@@ -12,11 +12,11 @@
 import math
 import networkx as nx
 
-from Star import Star
-from  TradeCalculation import TradeCalculation, NoneCalculation, CommCalculation, XRouteCalculation, \
+from PyRoute.Star import Star
```

Comment:
This is an interesting thing I've gone back and forth on a few times, and depends on how you run this code directly from the repo. Which I do. I'll have to figure out the command line setup to make this work again.